concent_api/utils/message_handler.py

```diff
@@ -0,0 +1,72 @@
+from functools                      import wraps
+
+from django.db                      import transaction
+from django.http                    import JsonResponse
+from django.http                    import HttpResponse
+from django.http                    import HttpResponseNotAllowed
+from django.conf                    import settings
+
+from golem_messages                 import dump
+from golem_messages.message         import Message
+
+from core.exceptions                import Http400
+
+from utils                          import logging
+
+
+def message_handler(view):
```

Comment:
Same here, we need more descriptive name, as it actually handles transactions, errors and responses. How about `handle_errors_and_responses` ? If you have better one feel free to replace it.

Reply:
Done